.gitignore

```diff
@@ -3,6 +3,7 @@
 !/bin/validate-json
 coverage
 .buildpath
+.idea
```

Comment:
This should be in the [global ignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)